src/parse.js

```diff
@@ -498,7 +498,11 @@ Parser.prototype.parseClassDef = function() {
             })
         };
     }
-    throw new Error('0x' + startOffset.toString(16) + ': ClassDef format must be 1 or 2.');
+
+    console.warn(`0x${startOffset.toString(16)}: Unknown ClassDef format ${format}.`);
```

Comment:
The error should probably let them know the font is likely corrupt so that they know to re-acquire it, instead of posting a bug report.